docs/pages/docs/core-concepts/caching.mdx

```diff
@@ -209,6 +227,11 @@ Luckily, you can control `turbo`'s cache fingerprinting (also known as "hashing"
 }
 ```
 
+<Callout type="info">
+  It is also possible to declare environment variables in the `dependsOn` config with a `$` prefix for
```

Comment:
Let's not present this in a "positive" framing ("it is also possible") and instead jump right into "is deprecated."